nucleus/annotation.py

```diff
@@ -814,24 +815,35 @@ class CategoryAnnotation(Annotation):
             label="dress",
             reference_id="image_1",
             taxonomy_name="clothing_type",
-            metadata={"dress_color": "navy"}
+            metadata={"dress_color": "navy"},
+        )
+
+        scene_category = CategoryAnnotation(
+            label="throwing",
+            reference_id="scene_1",
+            taxonomy_name="human_action",
+            metadata={"object_thrown": "baseball"},
+            is_scene=True,
```

Comment:
Are `reference_id`s globally unique or just per dataset @drakejwong ? In case it's globally unique we can get if it's a scene or not through a join with the dataset table. 

I agree with @rkaplan we should get rid of the `is_scene`. @tetranoir and I started with changing to an enum already but didn't complete it unfortunately. Will add a ticket for this, we need to clean this up.

Reply:
they are only unique per dataset. also a datasetitem and scene can share the same reference_id :( ongoing thread about this [here](https://scaleapi.slack.com/archives/C02A7VAKQ3C/p1656354203795129?thread_ts=1656352623.886209&cid=C02A7VAKQ3C)

totally agree, this should be an enum at least or some other paradigm. separately, we have a potential "api v1.0" workstream that will have some breaking changes but ultimately fix a lot of these user frictions

Reply:
Let's continue the convo [here](https://scaleapi.slack.com/archives/C02A7VAKQ3C/p1656352623886209)